atlantic-2/proposals/05-13-2023-no-slashing.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Update Slashing Fraction Proposal",
+  "description": "This proposal aims to update the slashing fraction for both double sign and missing signatures to zero.",
+  "changes": [
+    {
+      "subspace": "slashing",
+      "key": "SlashFractionDoubleSign",
+      "value": "0"
+    },
+    {
+      "subspace": "slashing",
+      "key": "SlashFractionDowntime",
+      "value": "0"
```

Comment:
@BrandonWeng should we also make the modification to oracle slashing as well here?